packages/expo/package.json

```diff
@@ -43,7 +43,8 @@
     "@clerk/clerk-react": "5.0.0-beta.30",
     "@clerk/shared": "2.0.0-beta.19",
     "base-64": "^1.0.0",
-    "react-native-url-polyfill": "2.0.0"
+    "react-native-url-polyfill": "2.0.0",
+    "tslib": "2.4.1"
```

Comment:
Why we downgraded to 2.4.1 here?

Reply:
To consolidate all the different `tslib` versions. I will drop this change and handle it in a separate PR